cues.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: default
+---
+
+# Cues
+
+## Introduction
+
+The `Cues Element` provides an index of certain `Cluster Elements` to allow for optimized seeking to absolute timestamps within the `Segment`. The `Cues Element` contains one or many `CuePoint Elements` which each MUST reference an absolute timestamp (via the `CueTime Element`), a `Track` (via the `CueTrack Element`), and a `Segment Position` (via the `CueClusterPosition` Element). Additional non-mandated Elements are part of the `CuePoint Element` such as `CueDuration`, `CueRelativePosition`, `CueCodecState` and others which can improve seeking performance in some Matroska players.
+
+## Recommendations
+
+The following recommendations are provided to optimize Matroska performance.
+
+- Unless Matroska is used as a live stream, it SHOULD contain a `Cues Element`. Note that some Matroska players may not be able to play a Matroska file without a `Cues Element`.
```

Comment:
I can't remember where I got this sentence, but I feel like I copied it in from some historic Matroska document. At any rate, I'll remove it.